client/sections.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-const fs = require( 'fs' );
+import fs from 'fs';
```

Comment:
This file is used by Webpack loader, and also is used in both server and client side code by both Webpack configs. That's why it's perfect to validate if ES6 is enabled properly.